docker_entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Both good arguments: let's leave it in the root, alongside the Dockerfile, named as-is. :+1: